src/Symfony/Component/HttpFoundation/EventStreamResponse.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation;
+
+/**
+ * Represents a streaming HTTP response for sending server events
+ * as part of the Server-Sent Events (SSE) streaming technique.
+ *
+ * @see ServerEvent
+ *
+ * @author Yonel Ceruto <open@yceruto.dev>
+ *
+ * Example usage:
+ *
+ *     return new EventStreamResponse(function () {
+ *         while (true) {
```

Comment:
We should maybe not use an infinite loop in the docs, because PHP is not designed for that and this will crash (timeout, memory leaks...) except if php.ini is tweaked.

Maybe that we could just send some events with a sleep between them then close the connection.

Reply:
agree, loop removed from the example